src/v2/Apps/Show/Components/ShowNavigationBanner.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { BoxProps } from "@artsy/palette"
+import { BackLink } from "v2/Components/Links/BackLink"
+import { ShowNavigationBanner_show } from "v2/__generated__/ShowNavigationBanner_show.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface ShowNavigationBannerProps {
+  show: ShowNavigationBanner_show
+}
+
+const ShowNavigationBanner: React.FC<ShowNavigationBannerProps & BoxProps> = ({
+  show,
+  ...rest
+}) => {
+  const { fair } = show
+
+  return (
+    <BackLink {...rest} to={`${fair?.href}/exhibitors`}>
+      Back to {fair?.name}
+    </BackLink>
+  )
```

Comment:
** thought**: maybe it's better not to render the BackLink at all, if not fair.name?